src/AppInstallerRepositoryCore/Microsoft/SQLiteIndex.h

```diff
@@ -146,6 +146,8 @@ namespace AppInstaller::Repository::Microsoft
         // Largely a utility function; should not be used to do work on behalf of the index by the caller.
         Utility::NormalizedName NormalizeName(std::string_view name, std::string_view publisher) const;
 
+        bool ValidateManifest(SQLite::Connection& connection, const Manifest::Manifest& manifest);

```

Comment:
See previous request for changes and new comment; expose the information through this interface, don't do the validation inside it.